src/Chainweb/Version.hs

```diff
@@ -1033,6 +1034,13 @@ chainweb218Pact Development = (>= 500)
 chainweb218Pact (FastTimedCPM g) | g == petersonChainGraph = (> 60)
 chainweb218Pact _ = (> 24)
 
+chainweb219Pact :: ChainwebVersion -> BlockHeight -> Bool
+chainweb219Pact Mainnet01 = (>= 3_774_423) -- 2023-06-02 00:00:00+00:00
+chainweb219Pact Testnet04 = (>= 3_299_753) -- 2023-06-01 12:00:00+00:00
+chainweb219Pact Development = (>= 550)
+chainweb219Pact (FastTimedCPM g) | g == petersonChainGraph = (> 70)
+chainweb219Pact _ = (> 26)
```

Comment:
The previous service date was "2023-06-01T00:00:00Z". Are we a day out for each of testnet and mainnet for a reason?

Reply:
Testnet fork is 12 hours after service date, mainnet fork is 24 hours after.

Reply:
👍

Reply:
Alright, makes sense, +1 on the dates